app/views/posts/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="new_post">
```

Comment:
- [ ] Good job by adding post form :+1:  But according to the project requirement you should also add  a form in the **views/comments/new.html.erb file** to make users able to create comments :smiley:  Please fix it in order to make your project work as expected :+1: